sw/device/lib/arch/device_sim_verilator.c

```diff
@@ -12,7 +12,8 @@
 const device_type_t kDeviceType = kDeviceSimVerilator;
 
 // Changes to the clock frequency or UART baud rate must also be reflected at
-// `hw/top_earlgrey/rtl/top_earlgrey_verilator.sv`.
+// `hw/top_earlgrey/rtl/top_earlgrey_verilator.sv` and
+// `test/systemtest/earlgrey/test_sim_verilator.py`.
```

Comment:
Um, maybe these comments aught to go on the baudrate itself...?

Reply:
They are relevant for the (peripheral and normal) clock frequencies as well as the baud rate.